docs/source/data/kedro_io.md

```diff
@@ -31,9 +31,9 @@ If you have a dataset called `parts`, you can make direct calls to it like so:
 parts_df = parts.load()
 ```
 
-However, we recommend using a `DataCatalog` instead (for more details, see [this section](../data/data_catalog.md) in the User Guide) as it has been designed to make all datasets available to project members.
+However, we recommend using a `DataCatalog` instead (for more details, see [the `DataCatalog` documentation](../data/data_catalog.md)) as it has been designed to make all datasets available to project members.
 
-For contributors, if you would like to submit a new dataset, you will have to extend `AbstractDataSet`. For a complete guide, please read [Creating a new dataset](../extend_kedro/custom_datasets.md).
+For contributors, if you would like to submit a new dataset, you will have to extend `AbstractDataSet`. For a complete guide, please read [the section on custom datasets](../extend_kedro/custom_datasets.md).
```

Comment:
you must extend `AbstractDataSet`. For a complete guide, please read the section on [custom datasets](../extend_kedro/custom_datasets.md).